web/massastation/src/pages/Store/Store.tsx

```diff
@@ -11,7 +11,7 @@ export function Store() {
     <>
       <div className="bg-primary text-f-primary pt-24">
         <h1 className="mas-banner mb-10"> {Intl.t('store.modules-banner')}</h1>
-        <div className="overflow-auto h-[65vh]">
+        <div className="overflow-auto h-full">
```

Comment:
The motivation behind this change is that with the h-[65] the hover does not have the right behavior see video : 
https://github.com/massalabs/station/assets/90157528/c2f85d66-576d-4908-9158-2bac06ced67d

Here is the correct behavior with h-full :
https://github.com/massalabs/station/assets/90157528/fbf26daa-2935-4bd2-97db-1da7f30f73ce

But I understand that it also creates instability and will try to search for a better alternative

Reply:
I changed it back to [65vh] because I was having inconsistent results, it would be great if some could checkout this branch and tell me if it works as expected @mazmassa  @Thykof

Reply:
Since Mario I know you are not a fan if the hover behavior, is there an alternative to displaying this error message that requires less rework ?